examples/DecayingTurbulence.ipynb

```diff
@@ -79,7 +79,7 @@
     }
    ],
    "source": [
-    "u = flow.units.convert_velocity_to_pu(lattice.u(simulation.f)).numpy()\n",
+    "u = flow.units.convert_velocity_to_pu(lattice.u(simulation.f)).cpu().numpy()\n",
```

Comment:
For improved readability and to leverage our library's built-in functionalities, I recommend using the lattice.convert_to_numpy() method. This method not only simplifies the code but also ensures compatibility with all devices.